recipes/realgud-debugger2

```diff
@@ -0,0 +1,3 @@
+(realgud-debugger2 :fetcher github :repo "rocky/realgud-ruby-debugger2"
```

Comment:
Thanks. Should now be fixed by [5c6dc28]

On Thu, Mar 3, 2016 at 2:49 AM, Steve Purcell <notifications@github.com>
wrote:

> In recipes/realgud-debugger2
> <https://github.com/melpa/melpa/pull/3647#discussion_r54845676>:
>
> > @@ -0,0 +1,3 @@
> > +(realgud-debugger2 :fetcher github :repo "rocky/realgud-ruby-debugger2"
>
> Here the realgud-debugger2 name doesn't match the name of realgud-rdb2.el
> -- the package name and main contained filename should match, so you
> probably want to rename this package file.
>
> Inside realgud-rdb2.el has (defgroup realgud ...) in it, which should
> presumably be (defgroup realgud-rdb2 ...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/melpa/melpa/pull/3647/files#r54845676>.
>